src/@types/File.ts

```diff
@@ -0,0 +1,27 @@
+export interface UrlFiles {
```

Comment:
this describes a single object so should not be plural. Guess `File` clashes with Node.js-native `File`?

Reply:
yeah, we might have some issue later